src/main/docker/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+services:
+    artemis-app:
+        image: ghcr.io/ls1intum/artemis
+        build:
+            context: ../../..
+            dockerfile: src/main/docker/Dockerfile
+        volumes:
+            - ../resources/config/application-dev.yml:/home/artemis/application-dev.yml:ro
+            - ../resources/config/application-artemis.yml:/home/artemis/application-artemis.yml:ro
+        environment:
+            _JAVA_OPTIONS: -Xmx5120m -Xms2560m
+            SPRING_PROFILES_ACTIVE: dev,bamboo,bitbucket,jira,artemis,scheduling,athene,local
```

Comment:
Could these profiles cause problems, if the services are note defined in the `application-X.yml`?

Reply:
yes.
We could add a variable here with these profiles as default.
Or users can create a `docker-compose.overwrite.yml` file.